datasets/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""
+This file contains a function to download and extract a dataset.
+"""
+import urllib.request
+import zipfile
+
+__all__ = ['import_dataset']
+
+DATASETS = {
+    'house prices uk': {
+        'url': r'https://github.com/datasets/house-prices-uk/blob/master/data/data.csv',
+        'path': r'./house_prices_uk.data.csv'
+    },
+    'breast cancer':
+        {'url': r'https://archive.ics.uci.edu/ml/machine-learning-databases/breast-cancer/breast-cancer.data',
+         'path': r'./breast-cancer.data'}
+}
+
+
+def import_dataset(name):
+    # TODO - finish implementing - if the data already exists don't import it again!
+    if name not in DATASETS:
+        raise ValueError("This dataset is not supported yet.")
+    url, local_path = DATASETS[name]['url'], DATASETS[name]['path']
+    print(f"Downloading from {url} to {local_path}...")
+    urllib.request.urlretrieve(url, local_path)
+    print("Download complete.")
```

Comment:
Need to add a check that the data was downloaded properly. Are hashes/CRCs available for the datasets?

Reply:
Maybe, I don't know. I don't think this is that necessary... You can just open these using notepad or a similar program, it should have the format of a regular csv file.  
In the future when we're going to import datasets for deep learning, we could use a hash, and I think there is.

Reply:
I've looked over the links, you're right. For now we'll leave it at that